src/UrlGenerator.php

```diff
@@ -0,0 +1,338 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Router\FastRoute;
+
+use Nyholm\Psr7\Uri;
+use Yiisoft\Router\Group;
+use Yiisoft\Router\Route;
+use Yiisoft\Router\RouteCollectorInterface;
+use Yiisoft\Router\RouteNotFoundException;
+use Yiisoft\Router\UrlMatcherInterface;
+use Yiisoft\Router\UrlGeneratorInterface;
+use FastRoute\RouteParser;
+
+use function array_key_exists;
+use function array_keys;
+use function implode;
+use function is_string;
+use function preg_match;
+
+final class UrlGenerator implements UrlGeneratorInterface
```

Comment:
The name is now does not align with `Yiisoft\Router\FastRoute\FastRoute`. How about renaming `Yiisoft\Router\FastRoute\FastRoute` to `Yiisoft\Router\FastRoute\Router`?

Reply:
Seems it makes sense.